### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by Naawel** [kaushaltrivedi/fast-bert#242]
Hello, 

Does anyone know on what loss function is fast-bert based ? 
Is it categorical cross entropy ?

Thanks
